docs/capi-aws.md

```diff
@@ -0,0 +1,177 @@
+# Cluster API - AWS
+
+In this guide, we show you how to use AWS for the worker plane with the k0smotron control plane.
+
+## Prerequisites
+
+We assume that a k0s cluster has already been installed and is ready for use. 
+The kubeconfig should be available in ~/.kube/config and the context should already be set to the k0s cluster. 
+
+Next to the cluster we will need the following tools installed on your system:
+
+1. clusterctl CLI
+    - https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl
+2. clusterawsadm CLI
+    - https://github.com/kubernetes-sigs/cluster-api-provider-aws#clusterawsadm
+
+Also, you need to have an AWS account and your AWS CLI configured with your credentials.
+
+### Prepare the infra provider
```

Comment:
I would remove this whole section, we don't want to maintain the instructions for this. Besides the current instructions only work for mac with m chips.